pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2370,4 +2370,11 @@ self: super: {
   } super.postgrest));
 
   html-charset = dontCheck super.html-charset;
+
+  # true-name-0.1.0.4 has been tagged, but has not been released to Hackage.
+  # Also, beyond 0.1.0.4 an additional patch is required to make true-name
+  # compatible with current versions of template-haskell
+  # https://github.com/liyang/true-name/pull/4
+  true-name = appendPatch ./patches/true-name.patch
```

Comment:
We try not to carry around patches in Nixpkgs if at all possible.

Could you fetch this patch with `fetchpatch`?  There should be a bunch of examples of that in this file.